input/metadata/properties/default.json

```
@@ -5071,329 +5156,274 @@
-          "minimum": 0
+          "description": "The type of handler used.",
+          "pattern": "^(AlleyScraper|ManualScraper|FlushSystem|ParlorCleaningHandler)$"
         },
-        "cleanings_per_day": {
-          "type": "number",
-          "description": "The number of cleanings a specified manure handler performs per day.",
-          "minimum": 0
+        "handler_type": {
+          "type": "string",
+          "description": "The type of handler used.",
+          "pattern": "^(AlleyScraper|ManualScraper|FlushSystem|ParlorCleaningHandler)$"
         },
-        "daily_tillage_frequency": {
+        "cleaning_water_use_amount": {
           "type": "number",
-          "description": "Applicable to 'tillage'-type handlers; the number of times per day that a specified compost bedded pack barn pack is tilled.",
-          "minimum": 0
+          "description": "Amount of cleaning water used per animal per day (L/animal/day).",
+          "minimum": 0.0,
+          "maximum": 100.0,
+          "default": 0.0
         },
         "cleaning_water_recycle_fraction": {
           "type": "number",
-          "description": "The fraction (0.XX) of cleaning water used by a specified handler that is from recycled (not fresh) water sources (unitless).",
+          "description": "Fraction of cleaning water that is from recycled (not fresh) water sources (unitless).",
           "minimum": 0.0,
-          "maximum": 1.0
+          "maximum": 1.0,
+          "default": 0.8
+        },
+        "use_parlor_flush": {
+          "type": "bool",
+          "description": "Indicator for if a parlor flush is used in addition to routine parlor water cleaning with fresh water."
         }
       }
     },
-    "manure_separator_configs": {
+    "separator": {
       "type": "array",
+      "description": "A list of all configurations for separator processors.",
       "properties": {
         "type": "object",
+        "description": "Configuration for a separator processor.",
         "name": {
           "type": "string",
-          "description": "Unique identifier to reference this solid liquid separator config in manure management scenario(s). This value should be unique and cannot match the name of any other manure processor.",
-          "pattern": "^(?!none$)(.*)$"
+          "description": "The unique identifier of the separator."
         },
-        "manure_separator_type": {
+        "processor_type": {
           "type": "string",
-          "description": "The type of mechanical solid-liquid separator.",
-          "pattern": "^(screw press|rotary screen)$"
+          "description": "The type of separator used.",
+          "pattern": "^(RotaryScreen|ScrewPress)$"
+        },
+        "separated_solids_dry_matter": {
+          "type": "number",
+          "description": "The dry matter content (percent DM by mass) of separated manure solids.",
```

```suggestion
          "description": "The dry matter fraction (0.XX percent DM by mass) of manure solids generated from solid liquid separation.",
```
@tomhuhh "Manure solids" is the common terminology for the solid fraction (dry solids + any moisture) resulting from solid liquid separation